content/diversity/_index.md

```diff
@@ -0,0 +1,20 @@
+# Diversity 
+
+As a [NumFocus supported project](https://numfocus.org), we abide by their [Code of Conduct and Diversity Statment](https://numfocus.org/code-of-conduct):  
+
+> NumFOCUS welcomes and encourages participation in our community by people of all backgrounds and identities. We are committed to promoting and sustaining a culture that values mutual respect, tolerance, and learning, and we work together as a community to help each other live out these values.
+
+> We have created this diversity statement because we believe that a diverse community is stronger, more vibrant, and produces better software and better science. A diverse community where people treat each other with respect has more potential contributors, more sources for ideas, and fewer shared assumptions that might hinder development or research.
+
+> Although we have phrased the formal diversity statement generically to make it all-inclusive, we recognize that there are specific identities that are impacted by systemic discrimination and marginalization. We welcome all people to participate in the NumFOCUS community regardless of their identity or background.
+
+We believe that having diversity and inclusion in our communtiy is the only way we can have a strong and vibrant community, we have and will countinue to make significant investments in making sure that we relentlessly push towards that goal. 
+
+Below you will find a breakdown of our past and current diversity and inclusion initatives. If you have any ideas, feedback, or the like, feel free to send and email to diveristy@julialang.org or drop by the [#diversity channel on Slack](https://slackinvite.julialang.org).
```

Comment:
@ViralBShah can we make a diversity@julialang.org email or shall we use an existing email for this?